src/signals/incident-management/containers/IncidentSplitContainer/components/IncidentSplitForm/__tests__/IncidentSplitForm.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+
+import { fireEvent, render } from '@testing-library/react';
+import { withAppContext } from 'test/utils';
+
+import { subcategoriesWithUniqueKeys as subcategories } from 'utils/__tests__/fixtures';
+import directingDepartmentList from 'signals/incident-management/definitions/directingDepartmentList';
+import parentIncidentFixture from '../../../__tests__/parentIncidentFixture.json';
+
+import IncidentSplitForm from '..';
+
+const mockHistoryPush = jest.fn();
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useHistory: () => ({ push: mockHistoryPush }),
+}));
+
+describe('IncidentSplitForm', () => {
+  const onSubmit = jest.fn();
+  const props = { parentIncident: parentIncidentFixture, subcategories, onSubmit };
+
+  it('should render correctly', () => {
+    const { queryAllByText } = render(withAppContext(<IncidentSplitForm {...props} />));
+    expect(queryAllByText(parentIncidentFixture.description)).toHaveLength(1);
+  });
+
+  it('should handle submit', async () => {
+    const { findByTestId, getByTestId } = render(withAppContext(<IncidentSplitForm {...props} />));
+
+    fireEvent.click(getByTestId('incidentSplitFormIncidentSplitButton'));
+    fireEvent.submit(getByTestId('incidentSplitFormSubmitButton'));
+
+    await findByTestId('incidentSplitForm');
+
+    expect(onSubmit).toHaveBeenCalledWith({
+      department: directingDepartmentList[0].key,
+      // WARNING: data does not match acc and that sucks because it is confusing..
```

Comment:
Can we make this comment more friendly? :-)

Reply:
Haha, yes. Is the comment talking about itself? Because it's quite 'confusing.' :P

Reply:
Hehe, was a not to myself, will remove it on next pull request ;)

Reply:
Removed in: SIG-3223-split-button-visibility